tools/distpartitioning/convert_partition.py

```diff
@@ -18,6 +18,36 @@ def create_dgl_object(graph_name, num_parts, \
     This function creates dgl objects for a given graph partition, as in function
     arguments. 
 
+    The input schema object is structure as follows: 
+    1. "nid" is a dictionary, which has the following structure. 
+        "ntype-name" : {
+            "format" : "csv", 
+            "data" : [
+                    [ <path-to-file>/ntype0-name-0, start_id, end_id], 
+                    [ <path-to-file>/ntype0-name-1 start_id, end_id)
+            ]
+        }
+
+    2. "eid" is a dictionary, which has the following structure
+        "etype-name" : {
+            "format" : "csv", 
+            "data" : [
+                    [ <path-to-file>/ntype0-name-0, start_id, end_id], 
+                    [ <path-to-file>/ntype0-name-1 start_id, end_id)
+            ]
+        }
+
+    The "nid" value is a dictionary, which described the type_nid that
+    should be assigned to nodes which are read from the corresponding nodes file. 
+
+    Similarly, compared to "nid" dictionary, "eid" dictionary is structure as well. 
```

Comment:
can you make the description more precise?

Reply:
eid dictionary is a structure similar to the nid dict.

Reply:
Done.